com.unity.render-pipelines.universal/Runtime/Passes/GBufferPass.cs

```diff
@@ -112,20 +112,20 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
 
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
-            CommandBuffer gbufferCommands = CommandBufferPool.Get();
-            using (new ProfilingScope(gbufferCommands, m_ProfilingSampler))
+            var cmd = renderingData.commandBuffer;
+            using (new ProfilingScope(cmd, m_ProfilingSampler))
             {
-                context.ExecuteCommandBuffer(gbufferCommands);
-                gbufferCommands.Clear();
+                context.ExecuteCommandBuffer(cmd);
+                cmd.Clear();
 
                 // User can stack several scriptable renderers during rendering but deferred renderer should only lit pixels added by this gbuffer pass.
                 // If we detect we are in such case (camera is in overlay mode), we clear the highest bits of stencil we have control of and use them to
                 // mark what pixel to shade during deferred pass. Gbuffer will always mark pixels using their material types.
                 if (m_DeferredLights.IsOverlay)
                 {
-                    m_DeferredLights.ClearStencilPartial(gbufferCommands);
-                    context.ExecuteCommandBuffer(gbufferCommands);
-                    gbufferCommands.Clear();
+                    m_DeferredLights.ClearStencilPartial(cmd);
+                    context.ExecuteCommandBuffer(cmd);
```

Comment:
I don't think these are needed as long as the ClearStencilPartial uses the same buf.